pde2e-image/lib/darwin/runner.sh

```
@@ -353,18 +399,18 @@ fi
-# Checkout Podman Desktop only if necessary
+# Checkout repository only if necessary
 if [[ "$extTests" -eq 1 ]] && [ -n "$podmanDesktopBinary" ] ; then
-    echo "Running ext. tests and podman Desktop binary is specified, skipping checkout for podman-desktop"
+    echo "Running ext. tests and binary is specified, skipping checkout for $repo"
 else
-    echo "Checking out Podman Desktop repository"
-    clone_checkout "podman-desktop" $fork $branch
-    cd "$workingDir/podman-desktop"
+    echo "Checking out $repo repository"
+    clone_checkout "$repo" "$fork" "$branch" "$gitProviderUrl"
+    cd "$workingDir/$repo"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)